src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  let total = 0;
+  const everyDay = 40;
+
+  if (days < 3) {
+    total = days * everyDay;
+  } else if (days >= 3 && days <= 6) {
```

Comment:
Ты уже проверила в предыдущем if что days больше трех, если дошла до этого if
```suggestion
  } else if (days <= 6) {
```